cmd/kubeadm/app/phases/markmaster/markmaster.go

```diff
@@ -31,7 +31,11 @@ func MarkMaster(client clientset.Interface, masterName string, taints []v1.Taint
 	glog.Infof("[markmaster] Marking the node %s as master by adding the label \"%s=''\"\n", masterName, constants.LabelNodeRoleMaster)
 
 	if taints != nil && len(taints) > 0 {
-		glog.Infof("[markmaster] Marking the node %s as master by adding the taints %v\n", masterName, taints)
+		taintStrs := []string{}
+		for _, taint := range taints {
```

Comment:
@liztio  This is a concrete example of e2e verification that should be added.  There are a bunch of others in this PR.